changelog/1015.feature.rst

```diff
@@ -0,0 +1 @@
+Add :attr:`attachment` to :class:`InteractionApplicationCommandCallbackData`
```

Comment:
I don't think this changelog is necessary, since `disnake.types` isn't documented